ion/integration_test.go

```diff
@@ -40,47 +40,51 @@ type ionItem struct {
 
 var binaryRoundTripSkipList = []string{
 	"allNulls.ion",
-	"bigInts.ion",
-	"clobWithNonAsciiCharacter.10n",
-	"clobs.ion",
-	"decimal64BitBoundary.ion",
-	"decimals.ion",
-	"float32.10n",
-	"floats.ion",
-	"intBigSize1201.10n",
-	"intBigSize13.10n",
-	"intBigSize14.10n",
-	"intBigSize16.10n",
-	"intBigSize256.10n",
-	"intBigSize256.ion",
-	"intBigSize512.ion",
-	"intLongMaxValuePlusOne.10n",
-	"localSymbolTableImportZeroMaxId.ion",
-	"nullDecimal.10n",
 	"nulls.ion",
-	"structWhitespace.ion",
-	"subfieldInt.ion",
-	"subfieldUInt.ion",
-	"subfieldVarInt.ion",
-	"subfieldVarUInt.ion",
-	"subfieldVarUInt15bit.ion",
-	"subfieldVarUInt16bit.ion",
-	"subfieldVarUInt32bit.ion",
-	"symbolEmpty.ion",
-	"symbols.ion",
-	"T2.10n",
-	"T3.10n",
-	"T5.10n",
-	"T7-large.10n",
-	"T9.10n",
+	"nullDecimal.10n",
 	"testfile22.ion",
-	"testfile23.ion",
-	"testfile31.ion",
-	"testfile35.ion",
-	"testfile37.ion",
-	"utf16.ion",
-	"utf32.ion",
-	"whitespace.ion",
+	"T5.10n", /*
+		"bigInts.ion",
```

Comment:
Do these tests no longer fail using the new `Reader`-based roundtrip test?

Reply:
Updated the list